source/infrastructure/lib/aws-instance-scheduler-stack.ts

```diff
@@ -463,11 +356,11 @@ export class AwsInstanceSchedulerStack extends cdk.Stack {
     * Instance scheduler custom resource reference.
     */
     let customService = new cdk.CustomResource(this, 'ServiceSetup', {
-      serviceToken: lambdaToDynamoDb.lambdaFunction.functionArn,
+      serviceToken: coreScheduler.lambdaFunction.functionArn,
       resourceType: 'Custom::ServiceSetup',
       properties: {
         timeout: 120,
-        config_table: cfnConfigTable.ref,
+        config_table: (coreScheduler.configTable.node.defaultChild as dynamodb.CfnTable).ref,
```

Comment:
NIT: If you're going to use this multiple times, I'd put it in a variable.

Reply:
it is a variable, but by extracting the lambda and dynamo code into a construct it was no longer exposed and I didn't feel like exposing it was appropriate for the logical boundaries I was trying to enforce for the construct. Open to ideas for how to improve this as I currently view it as a "just make it work" hack that allowed the refactor to be done non-destructively